### DISCUSSION TITLE
Curl Options

### DISCUSSION THREAD

**Discussion by mhuggins7278** [rest-nvim/rest.nvim#345] [Q&A] [answered]
I have a use case where I am calling an API that is behind an auth provider. The flow should be that calling the endpoint with an Authorization header with a "Bearer" Token results in that request getting sent to our auth provider to verify the token. The auth provider will redirect back along with a set-cookie and then my request should be able to access my endpoint. 

The issue I'm running into is that I can't seem to use the cookie-jar. How do I pass command line options to to curl using the extension.

**Comment by boltlessengineer** [comment, marked as answer]
You can’t pass curl arguments with rest.nvim (yet)
But you can manually set cookies in headers:

```
GET https://example.com
Set-Cookie: name=value;path=/
```

Also rest.nvim supports cookies since v3 release, so that use-case will work now.
